### DISCUSSION TITLE
Welcome to shields Discussions!

### DISCUSSION THREAD

**Discussion by PyvesB** [badges/shields#5238] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by abhigoyani** [comment]
I found this thread while browsing about highlight.

**Comment by Andre601** [comment]
I think this discussion should be pinned in case there are more discussions covering it up in the list.

**Comment by IFirstYou** [comment]
I think we should have to close this discussion

**Comment by orangmuda** [comment]
now is 2022 I think we should have to close this discussion

**Reply by calebcartwright** [reply]
There is no such thing as "closing" a GitHub Discussion

**Reply by PyvesB** [reply]
We could always unpin it or lock it. Probably doesn't matter too much either way. :)
